spec/services/solidus_bolt/accounts/detail_service_spec.rb

```diff
@@ -4,23 +4,24 @@
 
 RSpec.describe SolidusBolt::Accounts::DetailService, :vcr, :bolt_configuration do
   describe '#call', vcr: true do
-    subject(:api) { described_class.new(access_token: access_token) }
+    subject(:detail) { described_class.call(access_token: access_token) }
 
     context 'with wrong access_token' do
       let(:access_token) { 'Bolt Access Token' }
 
       it 'gives an error' do
-        expect{ api.call }.to raise_error(SolidusBolt::ServerError, 'This action is forbidden.')
+        expect{ detail }.to raise_error(SolidusBolt::ServerError, 'This action is forbidden.')
       end
     end
 
     context 'with correct access_token' do
       let(:access_token) { ENV['BOLT_ACCESS_TOKEN'] }
 
       it 'receives a successful response' do
-        expect(api.call).to match hash_including('profile')
-        expect(api.call).to match hash_including('addresses')
-        expect(api.call).to match hash_including('payment_methods')
+        api_call = detail
```

Comment:
You don't have to assign it to a variable. Do you mind removing it?

Reply:
fixed in 8da3ea2